cal/cal/helpers.py

```diff
@@ -88,18 +100,22 @@ def get_time_series(model, timezone='UTC', time_step='week', calendar_ids=None,
         # Change start date to beginning of month
         while start.day != 1:
             start = start - timedelta(days=1)
+        increment = relativedelta(months=1)
     start = start.replace(hour=0, minute=0, second=0, microsecond=0)
     # Convert back to UTC
     start = start.astimezone(pytz.utc)
     # Rollover takes care of events that overlap time periods
     rollover = 0
+    i = 0
+
+    # For moving average
+    data_point = 0
+    period_total = 0
+    # Change period to change number of data points for average
+    period = 5.0
```

Comment:
Change this to 30. See comment below for my justification for a 30 day moving average.